test/unit/math/dataGen.hpp

```diff
@@ -0,0 +1,53 @@
+/** Copyright 2019 Axel Huebl, Benjamin Worpitz
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/**
+ * @namespace test
+ * @brief Only contains fillWithRndArgs.
+ * @fn fillWithRndArgs
+ * @tparam Data The used Buffer-type.
+ * @param buffer The buffer that should be filled.
+ * @param size The size of the used buffer.
+ * @param range The Range, around Zero, for the data.
+ */
+
+namespace test
+{
+    template< typename Data >
+    auto fillWithRndArgs(
+        Data * buffer,
+        size_t size,
+        size_t range
+    ) -> void
+    {
+        std::random_device rd {};
+        std::default_random_engine eng { rd( ) };
+        std::uniform_real_distribution< Data > dist(
```

Comment:
this will generate unpredictable numbers each run and {the distribution} is [not platform portable](https://stackoverflow.com/questions/14840901/is-the-random-library-in-c11-portable).
I would like to know the numbers chosen for the test, so either we use hard-coded numbers or we might want to use seedable and platform-portable random numbers.

Reply:
+1 for a seed that is printed before the test is executed